doc/sphinx/whats-new/index.rst

```diff
@@ -13,18 +13,14 @@ This section describes the changes and improvements between different
 versions of Varnish, and what upgrading between the different versions
 entail.
 
-Varnish **$NEXT_RELEASE**
--------------------------
-
-**Note: These are working documents for a future release, with running
-updates for changes in the development branch. For changes in the
-released versions of Varnish, see the chapters listed below.**
+Varnish **7.4**
+---------------
```

Comment:
See what @nigoroll did for the previous releases, please drop the emphasis on `**7.4**`.

Reply:
Fixing